### PR TITLE
Only install uWSGI in the Docker image.

### PR DESCRIPTION
**Pull request by jchristgit**
As discussed in the dev channels.

Not sure why the psycopg2 hashes were updated.

### CODE REVIEW

File: docker/app/Dockerfile [hunk 1]
```diff
@@ -28,7 +28,8 @@ RUN rm -r /opt/bitnami/python/lib/python3.*/site-packages/setuptools* && \
     pip install --no-cache-dir -U setuptools
 
 RUN python3 -m pip install pipenv \
-    && python3 -m pipenv install --dev --system --deploy
+    && python3 -m pipenv install --dev --system --deploy \
```
Comment (jerbob) [inline docker/app/Dockerfile]: Do we need to install dev dependencies in this image?

Comment (jchristgit) [reply docker/app/Dockerfile]: No, will fix in separate PR.

Comment (jchristgit) [reply docker/app/Dockerfile]: fixed

**Review by jerbob — APPROVED** [review]
cool and ok

**Pull request merged**